servers/rendering/renderer_rd/shaders/blur.glsl

```diff
@@ -0,0 +1,228 @@
+/* clang-format off */
```

Comment:
I would also probably call the raster versions of effects something like blur_raster.glsl dof_raster.glsl, etc.

Reply:
blur_raster.glsl has both glow and dof so just one rename :)